django/core/jinja2/base.jinja

```diff
@@ -288,6 +288,21 @@
     <link rel="mask-icon" href="{{ static('favicon/safari-pinned-tab.svg') }}" color="#5bbad5">
     <meta name="msapplication-TileColor" content="#da532c">
     <meta name="theme-color" content="#ffffff">
+    {% block ogp_tags %}
+    <!-- Facebook Meta Tags -->
+    <meta property="og:url" content="https://www.comses.net/">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="CoMSES Net">
+    <meta property="og:description" content="CoMSES Net is an international open research community dedicated to fostering good practices for computational / agent based modeling.">
+    <meta property="og:image" content="https://complexity.asu.edu/sites/default/files/comses.jpg">
```

Comment:
these probably should refer to an internal static image.. I guess the current logo is generated by JS but it could live in core/static/images as well and be accessible via something like `{{ static("images/comses-logo.png") }}`. 

This can be done by copying `frontend/src/assets/images/comses-logo.png` to `django/core/static/images` and then replace all `og|twitter:image` references to the static invocation above